docs/usage/tutorials.md

```diff
@@ -0,0 +1,563 @@
+
+<details markdown="1">
+<summary>Expected files in database directory</summary>
+
+- `bracken`
+  - `hash.k2d`
+  - `opts.k2d`
+  - `taxo.k2d`
+  - `database100mers.kmer_distrib`
+  - `database150mers.kmer_distrib`
+
+</details>
+
+You can follow Bracken [tutorial](https://ccb.jhu.edu/software/bracken/index.shtml?t=manual) for more information.
+
+### Centrifuge custom database
+
+To build a custom Centrifuge database, a user needs to download taxonomy files, make a custom `seqid2taxid.map` and combine the fasta files together.
+
+In total, you need four components: a tab-separated file mapping sequence IDs to taxonomy IDs (`--conversion-table`), a tab-separated file mapping taxonomy IDs to their parents and rank, up to the root of the tree (`--taxonomy-tree`), a pipe-separated file mapping taxonomy IDs to a name (`--name-table`), and the reference sequences.
+
+An example of custom `seqid2taxid.map`:
+
+```
+ NC_001133.9 4392
+ NC_012920.1 9606
+ NC_001134.8 4392
+ NC_001135.5 4392
+```
+
+```bash
+centrifuge-download -o taxonomy taxonomy
+cat *.{fa,fna} > input-sequences.fna
+centrifuge-build -p 4 --conversion-table seqid2taxid.map --taxonomy-tree taxonomy/nodes.dmp --name-table taxonomy/names.dmp input-sequences.fna taxprofiler_cf
+```
+
+<details markdown="1">
+<summary>Expected files in database directory</summary>
+
+- `centrifuge`
+  - `<database_name>.<number>.cf`
+  - `<database_name>.<number>.cf`
+  - `<database_name>.<number>.cf`
+  - `<database_name>.<number>.cf`
+
+</details>
+
+For the Centrifuge custom database documentation, see [here](https://ccb.jhu.edu/software/centrifuge/manual.shtml#custom-database).
+
+### DIAMOND custom database
+
+To create a custom database for DIAMOND, the user should download and unzip the NCBI's taxonomy files and the input FASTA files.
+
+The download and build steps are as follows:
+
+```bash
+wget ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/taxdmp.zip
+unzip taxdmp.zip
+
+## warning: large file!
+wget ftp://ftp.ncbi.nlm.nih.gov/pub/taxonomy/accession2taxid/prot.accession2taxid.FULL.gz
+
+## warning: takes a long time!
+cat ../raw/*.faa | diamond makedb -d testdb-diamond --taxonmap prot.accession2taxid.FULL.gz --taxonnodes nodes.dmp --taxonnames names.dmp
+
+## clean up
+rm *dmp *txt *gz *prt *zip
+```
+
+<details markdown="1">
+<summary>Expected files in database directory</summary>
+
+- `diamond`
+  - `<database_name>.dmnd`
+
+</details>
+
+A detailed description can be found [here](https://github.com/bbuchfink/diamond/wiki/1.-Tutorial)
+
+### Kaiju custom database
+
+To build a kaiju database, you need three components: a FASTA file with the protein sequences ,the NCBI taxonomy dump files, and you need to define the uppercase characters of the standard 20 amino acids you wish to include.
+
+> ⚠️ The headers of the protein fasta file must be numeric NCBI taxon identifiers of the protein sequences.
```

Comment:
```suggestion
:::warning
The headers of the protein fasta file must be numeric NCBI taxon identifiers of the protein sequences.
:::
```

Reply:
Ah I forgot to go through the database stuff, will do that now